CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ CHANGELOG
 * "BF": Bug fix
 
 ---
+* 0.5.0
+  * PHP 7.2 is minimum supported version
+  * BC: `Ratchet\Session\SessionProvider` requires `Ratchet\Session\OptionsHandler`
```

Comment:
Technically not a BC break as the additional parameter is optional and will default to the INI handler which has the same behaviour as before.